### DISCUSSION TITLE
NEED HELP

### DISCUSSION THREAD

**Discussion by subitme** [sherlock-project/sherlock#1764] [Q&A]
How can I used this on my phone. Is it phone compatible, and also I need someone help in hacking someone's social account. Thank you susu

**Comment by subitme** [comment]
Anyone please 🥺 help me out

**Comment by MaazAr** [comment]
https://www.learntermux.tech/2020/01/install-and-use-sherlock-tool-for-android-in-termux-2020.html

Follow this guide to set up and use sherlock on your android device.

**Reply by Junubian** [reply]
How about on iPhone

**Comment by subitme** [comment]
Needed help to hack someones phone for me.

**Reply by ausward** [reply]
## Pretty sure if anyone was to help with that it would be against Github's TOS

**Comment by g4cg4c** [comment]
someone pls send me python download link

**Reply by ausward** [reply]
[-_-](https://www.python.org/)
